README.rst

```diff
@@ -1,6 +1,9 @@
 Raiden Network
 ==============
 
+.. image:: https://img.shields.io/badge/python-3.6-blue.svg
```

Comment:
Could you replace this with the URL that automatically extracts the supported versions from our PyPI packages: `https://img.shields.io/pypi/pyversions/raiden.svg`?